### PR TITLE
refs task #26425 manage users

### PR DESCRIPTION
**Pull request by vuchuc789**
## Related Tickets:
- [#26425](https://edu-redmine.sun-asterisk.vn/issues/26425)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by vuchuc789** [comment]
ready

File: resources/views/home.blade.php [hunk 1]
```diff
@@ -1,23 +1,34 @@
...
+                <div class="form-group">
+                    <label for="profile-name">Title</label>
+                    <input type="text" class="form-control" id="profile-name" name="title" value="{{ $user->title }}">
+                </div>
+                <div class="form-group">
+                    <label for="profile-name">Description</label>
+                    <textarea class="form-control" id="profile-name" name="description">{{ $user->description }}</textarea>
+                </div>
+                <button type="submit" class="btn btn-primary">Update</button>
+            </form>
```
Comment (huyenttm-0901) [inline resources/views/home.blade.php]: i18n

Comment (vuchuc789) [reply resources/views/home.blade.php]: Em đã sửa ạ.

**Comment by vuchuc789** [comment]
ready

**Pull request merged**